### DISCUSSION TITLE
MusicXML contribution?

### DISCUSSION THREAD

**Discussion by mrehler** [madrisan/open-scores#11] [Ideas]
I don't work with Lilypond myself, but I'd love to contribute a score or two over my summer break. Do you do all the Lilypond by hand? Would you accept a pull request of something done with [xml2ly](https://www.nongnu.org/xml2ly/)? Any scores you'd like as part of this that you're not inclined to do yourself?

(I work with Dorico primarily)

**Comment by madrisan** [comment]
Hi, thanks for your interest in this project!
Yes, it's an open repository, so contributions are definitely welcomed.

And yes, up until now I have manually written the lilypond partitions of this repository.
Sometimes I found some partial source available in other sites, which I then integrated with the missing parts, or as in the case of the Art of Fugue by J.S. Bach I based my version on an available string one.
The partitions I insert are those I play myself or I'm interesting in.

I didn't know xml2ly and Dorico neither.
Its result can be used in the worst case as a basis for creating the final lilypond source.
Maybe with the help of a reformatting script I can write.
The boring part is always entering all the notes...

Do you have already an idea of what you would like to add?
Please let me known.

**Comment by madrisan** [comment]
> Any scores you'd like as part of this that you're not inclined to do yourself?

I forgot to answer to your question.
I only send the partitions that interest me, especially when I can't find one of graphic quality available on the internet.
You can add what you like. It would be great to have a contributor already :)

My TODO list (but  may change):
 * J.S. Bach: Partita n.6
 * J.S. Bach: Well-Tempered Clavier
 * Brahms: Sonata n.3

**Comment by madrisan** [comment]
Ping? :)
